package.json

```diff
@@ -2,30 +2,6 @@
   "name": "ember-cli-workbox",
```

Comment:
You may want to take a look at v2 package format for addons https://emberjs.github.io/rfcs/0507-embroider-v2-package-format.html